src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -256,6 +256,13 @@ def __init__(
         num_input_quant_stubs = num_inputs + len(self.kwarg_input_names)
 
         self.forward_fn = forward_fn
+        # Add weight qconfig to forward_fn (in case it has weights)
+        qconfig_ = get_qat_qconfig(qproperties)
+        qconfig = torch_quantization.QConfig(
```

Comment:
Since qconfig_ is already of type torch_quantization.QConfig, would it be simpler just to set its activation to identity?

Reply:
Attempted that and the activation cannot be changed after it's defined: AttributeError: can't set attribute

Reply:
ok, thanks for trying